resolwe_bio/docker_images/rnaseq/assets/spikeins_qc_template.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
Will these files be downloaded as zip usually, or will they be viewed directly on website?

Reply:
Ideally we should be able to open them in a separate tab. Should we change the `report` output  type from `basic:file` to `basic:url:view`?

Reply:
@jkokosar ?

Reply:
Opening this kind of reports in separate tab is always preferred over downloading.

@mstajdohar @JureZmrzlikar I think the type for the output field should be `basic:file:html`

Reply:
Discussion from daily:
- direct view link in `basic:file:html` is fine
- direct download link in `basic:file`/`basic:file:html` will **break external** css, fonts, and images
- downloading all files as zip should work

Workaround
- we can add a zipped output
  - can download when viewing a Sample or Data object
  - can't view in new tab
- we can add a hidden `basic:file:html` output and add `Open report` column in Sample Table
  - can open in new tab